tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -292,6 +292,7 @@ private static void GenerateMainProject(ApiMetadata api, string directory, HashS
                 // Build-related properties
                 new XElement("Version", api.Version), // TODO: Version, or VersionPrefix/VersionSuffix?
                 new XElement("TargetFrameworks", targetFrameworks),
+                new XElement("LangVersion", "latest"),
```

Comment:
Do we want to go for latest, or explicitly 7.1? The latter means we don't get broken implicitly when 7.2 comes out, if there are any breaking changes... but it means we need to go through this again. I'm in two minds.